js-intermediate.js

```diff
@@ -63,10 +111,46 @@ var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
 
+const midLetter = (string) => {
+    var position
+    var length
+    if(string.length % 2 == 1) {
+        position = string.length / 2
+        length = 1
+    } else {
+        position = string.length / 2 - 1
+        length = 2
+    }
+
+    return string.substring(position, position + length)
+}
+
+
+
+console.log(midLetter(middleLetters1))
+console.log(midLetter(middleLetters2))
+
+
 
 
 // 7. Write a program to get the area of a sphere using object classes. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
 
+var sphereRadius = { 
+    sphere1: 100 , 
+    sphere2: 20 ,
+    sphere3: 5
+}
+
+
+const areaOfSphere = (r) =>{
+    return 4*Math.PI*r**2
+    
+}
+
+console.log(areaOfSphere(sphereRadius.sphere1))
+console.log(areaOfSphere(sphereRadius.sphere2))
+console.log(areaOfSphere(sphereRadius.sphere3))
+
 
```

Comment:
This totally works, but the goal was to create a class and create spheres that are instances of the class (objects).